app/controllers/api/v1/events_controller.rb

```diff
@@ -25,54 +23,54 @@ def group_events
     end
 
     def add_movie
-      event = Event.find(params[:event_id])
+      event_1 = Event.find(params[:event_id])
```

Comment:
@Owenhorner ok I thought I could get a copy of your branch and also make changes to it but alas I can't. What I did before `git push` resulted in git asking for your credentials, was to change the `user` getter to all references to it, to `current_user` (and following that, `current_user_params'. 

I've changed the JS side so that it's passing in a post request to show rating so that can basically read
```
def show_rating
      render json: { rating: event.rating_for(user), average: event_1.average_rating }
    end
```

We'll need a new `user` getter:
```
def user
      @user ||= User.find(user_params)
    end

    def user_params
      params.require(:user).permit(:id)
    end
```